pallets/membership/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "pallet-membership"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
```

Comment:
Should we mention Dhiway instead of Parity?

Reply:
This is a pallet fork with minimum changes from our side. Hence retained the copyright and license.